### PR TITLE
print() is a function in Python 3

### PR DESCRIPTION
**Pull request by cclauss**
This PR should enable us to run the benchmarks under both Python 2 and Python 3.

### CODE REVIEW

**Comment by BSBandme** [comment]
Thanks for the PR to support python3 in benchmark1
But I think we need to do some more work. e.g. change 'open(...)' to 'open(..., "rb")', change the python/python_benchmark_messages.cc to work with python3-dev. 

Would you please make python3 work well for python benchmark?

**Comment by cclauss** [comment]
@BSBandme Let’s make the changes that you suggest under other PRs.  The focus of this PR was to fix the print statements, not to solve all Python 3 issues.

**Comment by BSBandme** [comment]
Asking Jie for python style reviewing @anandolee

**Comment by xfxyjwf** [comment]
@anandolee Jie, please take a look at this python change.

File: benchmarks/util/run_and_upload.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+from __future__ import absolute_import
 import argparse
 import os
 import re
 import copy
 import uuid
 import calendar
 import time
-import big_query_utils
+from . import big_query_utils
```
Comment (anandolee) [inline benchmarks/util/run_and_upload.py]: Why this change is related?

Comment (cclauss) [reply benchmarks/util/run_and_upload.py]: Python 2 tolerates relative imports but Python 3 does not without the leading dot.
